pgjdbc/src/test/java/org/postgresql/test/README.md

```diff
@@ -146,94 +119,17 @@ For example, in the comments you can explain where a certain test
 behaviour or the intended implementation of a feature?
 
-7 Example (incomplete)
-----------------------
-package org.postgresql.test.jdbc2;
-
-import static org.junit.Assert.assertNotNull;
-
-import org.postgresql.test.TestUtil;
-
-import org.junit.After;
-import org.junit.Before;
-import org.junit.Test;
-
-import java.sql.*;
-
-/*
- * Test case for ...
- */
-public class FooTest {
-
-    private Connection con;
-    private Statement stmt;
-
-    @Before
-    public void setUp() throws Exception {
-        con = TestUtil.openDB();
-        stmt = con.createStatement();
-
-        // Drop the test table if it already exists for some
-        // reason. It is not an error if it doesn't exist.
-        try {
-            stmt.executeUpdate("DROP TABLE testfoo");
-        } catch (SQLException e) {
-             // Intentionally ignore. We cannot distinguish
-             // "table does not exist" from other errors, since
-             // PostgreSQL doesn't support error codes yet.
-        }
-
-        stmt.executeUpdate(
-           "CREATE TABLE testfoo(pk INTEGER, col1 INTEGER)");
-        stmt.executeUpdate("INSERT INTO testfoo VALUES(1, 0)");
-
-        // You may want to call con.setAutoCommit(false) at
-        // this point, if most tests in this test case require
-        // the use of transactions.
-    }
-
-    @After
-    public void tearDown() throws Exception {
-        con.setAutoCommit(true);
-        if (stmt != null) {
-            stmt.executeUpdate("DROP TABLE testfoo");
-            stmt.close();
-        }
-        if (con != null) {
-              TestUtil.closeDB(con);
-        }
-    }
-
-    @Test
-    public void testFoo() {
-        // Use the assert methods in import org.junit.Assert
-        // for the actual tests
-
-        // Just some silly examples
-        assertNotNull(con);
-        if (stmt == null) {
-        	fail("Where is my statement?");
-        }
-    }
-
-    @Test
-    public void testBar() {
-    	// Another test.
-    }
-}
-
-8 ssltests
+7 SSL tests
 ----------------
-- requires ssl to be turned on in the database 'postgresql.conf ssl=true'
-- pg_hba.conf requires entries for hostssl, and hostnossl
+- requires SSL to be turned on in the database `postgresql.conf ssl=true`
+- `pg_hba.conf` requires entries for `hostssl`, and `hostnossl`
 - contrib module sslinfo needs to be installed in the databases
-- databases certdb, hostdb, hostnossldb, hostssldb, and hostsslcertdb need to be created
+- databases `certdb`, `hostdb`, `hostnossldb`, `hostssldb`, and `hostsslcertdb` need to be created
 
 
-9 Running the JDBC 2 test suite from Sun against PostgreSQL
+8 Running the JDBC 2 test suite against PostgreSQL
 ------------------------------------------------------------
-Download the test suite from
-http://java.sun.com/products/jdbc/jdbctestsuite-1_2_1.html
+Download the [JDBC test suite](http://java.sun.com/products/jdbc/jdbctestsuite-1_2_1.html). 
```

Comment:
I would think this link is dead

Reply:
I can get to that link.

Reply:
well it doesn't actually go got the jdbctestsuite any more, but I can fix this later